source/threads.tex

```diff
@@ -1338,7 +1338,7 @@
 \effects
 The new thread of execution executes
 \begin{codeblock}
-invoke(auto(std::forward<F>(f)), auto(std::forward<Args>(args))...)  // see \ref{func.invoke}
+invoke(auto(std::forward<F>(f)), auto(std::forward<Args>(args))...) // see \ref{func.invoke}
```

Comment:
This is wrong. See https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#formatting-declarations-and-definitions.